### DISCUSSION TITLE
9月21日的代码，下载文件程序崩溃

### DISCUSSION THREAD

**Discussion by wangtao0505** [CefView/QCefView#439] [Q&A] [answered]
bfa66417191d44ed888a9fa4f1e4330720d63d4c版本，使用自定义下载框，下载文件时候崩溃到libcef.dll里面了。

**Comment by tishion** [comment]
先降级CEF试试把

**Reply by wangtao0505** [reply]
使用113版本的cef之后可以正常下载，但是触屏电脑滑动网页时候崩溃。
之前就是想使用触屏效果，更新了最新的代码。
谁知道更新之后，触屏好用了，但是下载崩溃了。郁闷~~~~

大佬辛苦！！

**Reply by tishion** [reply, marked as answer]
已经修了
